### DISCUSSION TITLE
Welcome to NoitaMP Discussions!

### DISCUSSION THREAD

**Discussion by Ismoh** [Ismoh/NoitaMP#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sikerow** [comment]
i cant understand how to connect to a server. me and my friend have the same ip and port typed in. we have tried with a password and without a password. when we make a new world with multiplayer we both feel like the multiplayer mod should pop up but it wont. so we both create a new multiplayer world and when we have entered the world we cant see eachother or nothing.

**Reply by Ismoh** [reply]
Hey Mate,
I am really happy to see someone is testing this out, but I also unfortunately have to tell you that NoitaMP is still work in progress.
I am not sure, how you downloaded NoitaMP, I assume you downloaded a tag or master, which aren't playable atm.
I am currently working on #48 which is more or less functional, but clients fps are dropping soo bad, like NoitaMP does nomnomnom on client.

The only thing I can provide right now, is this specific branch for you guys, but dont expect fun!
If you can wait and be patient, there will be a working release (not a tag) in future!

Besides that NoitaMP is working like that:
- (in this case:) choose branch 48: https://github.com/Ismoh/NoitaMP/tree/48-developer-want-to-rework-client-and-server-modulesclasses
- click on download
- put mods folder of download into Noitas mods folder
- start Noita, go to mods and enable unrestricted mode, because NoitaMP is including networking dlls.
- start a new game with NoitaMP on the left menu
- on the bottom left you should see [+NoitaMP]
- click on it
- make sure to forward the typed in port, see https://portforward.com/

Regarding mod settings, you can have a look here: https://ismoh.github.io/NoitaMP/
Make sure to disable all debug settings, although those should be disabled by default (those settings will also nomnomnom fps)

If you have questions, don't hesitate to ask here!

And in addition I really should mention in the ReadMe that NoitaMP is still WIP!
When there is a playable version, I'll let you know!

Have a good one!

**Reply by LastChanse** [reply]
How connect to server using [+NoitaMP] menu?
Hi, I am download mod from branch 48 (https://github.com/Ismoh/NoitaMP/commit/7262b09f37c62013f6efd0c15c69ae571745f20b). But i don't know how connect to started server NoitaMP.
I see green triangle and I am press it, then server starting. I am open new window Noita and cant find connect to server in [+NoitaMP] menu.
![Безымянный](https://user-images.githubusercontent.com/91983973/234120733-f6292f61-da18-49b8-8eea-a976e7ce7bca.png)

![Снимок экрана 2023-04-25 002116](https://user-images.githubusercontent.com/91983973/234119675-b4a55095-caf1-456d-a875-91b4bd427b36.jpg)

**Reply by Ismoh** [reply]
> but I also unfortunately have to tell you that NoitaMP is still work in progress.

**and it's not working atm!**

**Reply by LastChanse** [reply]
But how it work on other PC? Not relise versions can isn't working and we need wait first release?
NoitaMP demonstration:
https://www.youtube.com/watch?v=drOVhoMa7UY

**Reply by LastChanse** [reply]
Oh no, I realized my mistake. I apologize for my incompetence. As I understood on the video, another mod, which is even more raw (

**Reply by LastChanse** [reply]
> > но я также, к сожалению, должен сказать вам, что работа над NoitaMP все еще продолжается.
> 
> **и это не работает банкомат! **

Is it possible to join you in the Discord channel about NoitaMP mod? If so, where can I find a link to the channel?

**Reply by Ismoh** [reply]
> But how it work on other PC? Not relise versions can isn't working and we need wait first release? NoitaMP demonstration: https://www.youtube.com/watch?v=drOVhoMa7UY

Whatever is shown in the video is *not* NoitaMP.

Discord: https://github.com/Ismoh/NoitaMP#tldr

You unfortunatelly just have to wait :(
